app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] Please fix this error occurs when we start the app. The page should render the sign_in page for us to save a current_user to our database. And continue to use the app. Tailwind wasn't a requirement for this milestone, if you can't make it please remove it from your project.
![image](https://github.com/Ridwanullahi-code/BlogApp/assets/91486791/a4c22f37-ae8d-4450-91b7-5805af152db6)
![image](https://github.com/Ridwanullahi-code/BlogApp/assets/91486791/daca2315-dc9a-4eb0-8c07-e726c4e69589)